### PR TITLE
update python and django version

### LINKED ISSUE #87: upgrade Django
To latest stable version of Django with dependencies:
Python
Django
DjangoCMS
all other dependencies

### PR DESCRIPTION
**Pull request by tinashechiraya**
Fixes https://github.com/kartoza/miniSASS/issues/87

Upgraded to django version 4.2
Upgraded python version to 3.11
integrated the frontend
integrated the monitor app from the old code
integrated the authentication app 
removed most of unused code

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 6 min 42 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 21 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 25 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 54 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 13 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 21 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 28 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 29 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 20 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 43 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 20 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 38 sec

File: django_project/minisass/settings.py [hunk 1]
```diff
@@ -85,215 +68,108 @@
...
-}
-
 # email settings
-EMAIL_HOST = os.getenv('SMTP_HOST')
-EMAIL_PORT = os.getenv('SMTP_PORT')
-DEFAULT_FROM_EMAIL = os.getenv('SMTP_EMAIL')
-EMAIL_HOST_USER = os.getenv('SMTP_HOST_USER')
-EMAIL_HOST_PASSWORD = os.getenv('SMTP_HOST_PASSWORD')
-EMAIL_USE_TLS = os.getenv('SMTP_EMAIL_TLS')
+# EMAIL_HOST = os.getenv('SMTP_HOST')
```
Comment (dimasciput) [inline django_project/minisass/settings.py]: @tinashechiraya why do you commented these? you can use `os.environ.get('SMTP_HOST', '')` instead.

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 55 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 6 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 30 sec

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 43 sec

**Pull request merged**